components/Article/ArticlePage.js

```diff
@@ -0,0 +1,89 @@
+                                {READ_MORE}
+                                ...
+                            </a>
+                        </p>
+                    </div>
+                    <div className="article-content">
+                        <p className="article-content-header">What Should be in a First Aid kit?</p>
+                        <p className="article-read">
+                            {ARTICLE_READ}
+                            <a className="article-content-a" href="/articlePage">
+                                {READ_MORE}
+                                ...
+                            </a>
+                        </p>
+                    </div>
+                </div>
+                <div className="content-wrapper">
+                    <div className="article-content article1">
+                        <p className="article-content-header">Why First Aid?</p>
+                        <p className="article-read">
+                            {ARTICLE_READ}
+                            <a className="article-content-a" href="/articlePage">
+                                {READ_MORE}
+                                ...
+                            </a>
+                        </p>
+                    </div>
+                    <div className="article-content">
+                        <p className="article-content-header">What Should be in a First Aid kit?</p>
+                        <p className="article-read">
+                            {ARTICLE_READ}
+                            <a className="article-content-a" href="/articlePage">
+                                {READ_MORE}
+                                ...
+                            </a>
+                        </p>
+                    </div>
+                </div>
+                <div className="content-wrapper">
+                    <div className="article-content article1">
+                        <p className="article-content-header">Why First Aid?</p>
+                        <p className="article-read">
+                            {ARTICLE_READ}
+                            <a className="article-content-a" href="/articlePage">
+                                {READ_MORE}
+                                ...
+                            </a>
+                        </p>
+                    </div>
+                    <div className="article-content">
+                        <p className="article-content-header">What Should be in a First Aid kit?</p>
+                        <p className="article-read">
+                            {ARTICLE_READ}
+                            <a className="article-content-a" href="/articlePage">
+                                {READ_MORE}
+                                ...
+                            </a>
+                        </p>
+                    </div>
+                </div>
+                <p className="pagination"> 1  |  2  |  3  |  4  |  ...</p>
```

Comment:
use antd list component to handle this list of contents and the pagination logic

Reply:
Done

Reply:
Will do that once I start connecting it to the api directly